lib/grape/swagger/jsonapi/resources.rb

```diff
@@ -1,15 +1,17 @@
-require "grape-swagger"
+require 'grape-swagger'
 
 # Only require the bits of Rails that jsonapi-resources uses
-require "action_controller"
-require "rails"
+require 'action_controller'
+require 'rails'
+require 'byebug'
```

Comment:
This will fail in production as `byebug` isn't in the bundle.

`require 'byebug' rescue LoadError nil` I think will fix it

Reply:
Oops. That should not be there.

Reply:
Gone